sqlstore/src/main/resources/db/migration/V8001__changelogs.sql

```diff
@@ -0,0 +1,188 @@
+ALTER TABLE BoxChangeLog ADD boxChangeLogId bigint(20) PRIMARY KEY AUTO_INCREMENT FIRST;
```

Comment:
We've been putting the other migrations in `V8900__hibernatize_everything` but I don't think it'll harm anything to have these in a different file.